docs/hermite.rst

```diff
@@ -0,0 +1,60 @@
+.. role:: raw-latex(raw)
+   :format: latex
+
+.. role:: html(raw)
+   :format: html
+.. _hermite:
+
+
+Multidimensional Hermite polynomials
+====================================
+.. sectionauthor:: Nicolás Quesada <nicolas@xanadu.ai>
+
+In this section we study the multidimensional Hermite polynomials. These polynomials where originally introduced by C. Hermite in 1865. Mizrahi :cite:`mizrahi1975generalized` provides an exhaustive reference on these subject. We however, follow the succinct treatment of Kok and Braunstein :cite:`kok2001multi`.
+
+In the next section, where we discuss Gaussian states we will explain how these polynomials relate to hafnians and loop hafnians. For the moment just let us introduce them and study their formal properties.
+
+Generating function definition
+******************************
+Given two complex vectors :math:`\alpha,\beta \in \mathbb{C}^\ell` and a symmetric matrix :math:`\mathbf{B} = \mathbf{B}^T \in \mathbb{C}^{\ell \times \ell}`
+
+.. math::
+	G_B(\alpha,\beta) = \exp\left( \alpha \mathbf{B} \beta^T - \tfrac{1}{2}\beta \mathbf{B} \beta^T\right) = \sum_{\mathbf{m} \geq \mathbf{0}} \prod_{i=1}^{\ell} \frac{\beta_i^{n_i}}{n_i} H_{\mathbf{m}}^{(\mathbf{B})}(\alpha),
+
+where the notation :math:`\mathbf{m} \geq 0` is used to indicate that the sum goes over all vectors  in :math:` \mathbb{N}^{\ell}_0`, the set of vectors of nonnegative integers of size :math:`\ell`. This generating function provides an implicit definition of the multidimensional Hermite polynomials.
+It is also straightforward to verify that :math:`H_{\mathbf{0}}^{(\mathbf{B})}(\alpha) = 1`.
+
+In the one dimensional case, :math:`\ell=1`, one can compare the generating function above with the ones for the "probabilists' Hermite polynomials" :math:`He_n(x)` and "physicists' Hermite polynomials" :math:`H_n(x)` to find
+
+.. math::
+	He_n(x) = H_{n}^{([1])}(x), \\
+	H_n(x) = H_{n}^{([2])}(x).
```

Comment:
Just noticed that this page contains a mixture of tabs and spaces; need to convert it all to spaces (can do this automatically in sublime)

Reply:
Happy to change it; does that make a difference when you compile the document?

Reply:
I'm not entirely sure, but I know that ReST can sometimes get confused by mixing spaces with tabs, so best practice to keep it standardized